jenkins-scripts/dsl/_configs_/Globals.groovy

```diff
@@ -63,12 +61,12 @@ class Globals
    // Main CI platform
    static ArrayList get_ci_distro()
    {
-    return [ 'xenial' ]
+    return [ 'bionic' ]
    }
 
    static ArrayList get_abi_distro()
    {
-     return [ 'xenial' ]
+     return [ 'bionic' ]
```

Comment:
I'd prefer to test and merge the change to `get_ci_distro` and `get_abi_distro` separately, since we can do that independently, and it may take a fair bit of testing

Reply:
sounds good, reverted 098293a

Reply:
ABI got in in https://github.com/ignition-tooling/release-tools/pull/470